runtime/onert/backend/cpu/ops/ConvolutionLayer.cc

```diff
@@ -164,6 +190,10 @@ void ConvolutionLayer::run()
   {
     convQuant8();
   }
+  else if (_input->data_type() == OperandType::QUANT_INT8_ASYMM)
```

Comment:
How about changing data type to `QUANT_INT8_SYMM_PER_CHANNEL`?

Reply:
@ragmani `input` is asymmetric.

Reply:
I see.

Reply:
@ragmani Thank you for pointing out `QUANT_INT8_SYMM_PER_CHANNEL`.

@hseok-oh, @ragmani I think `QUANT_INT8_SYMM_PER_CHANNEL` can (or should) be removed from onert IR.

Reply:
I think the type should not be removed. The weights may have the type with a param called as quantized dimension(channel index).

Reply:
> I think the type should not be removed. The weights may have the type with a param called as quantized dimension(channel index).

@ragmani I know the weights may have per-channel quantization params. I think `QUANT_INT8_SYMM_PER_CHANNEL` is introduced from Android NNAPI, which was introduced at API level 29 to represent per-channel quantization since prior type does not contain per-channel scales and zeropoints. However, onert IR can represent per-channel quantization without dedicated type as tflite does. We may create an issue if we need further discussion. It is also related to other types that has `_ASYMM`, `_SYMM`, ....

Reply:
`QUANT_INT8_SYMM_PER_CHANNEL` is used some backends such as ACL. If onert does not have that type, How are you going to get onert to support that type thought the weights may have per-channel quantization params?

Reply:
@ragmani I did not considered `ACL` backend. I've found `ARMComputeLibrary` uses `QSYMM8_PER_CHANNEL` type for channel-wise quantization tensor.